### DISCUSSION TITLE
Welcome to docker-firefox Discussions!

### DISCUSSION THREAD

**Discussion by jlesage** [jlesage/docker-firefox#182] [Announcements]
## 👋 Welcome!
Discussions has been enabled for this project.  This is a place to connect with other members of the community.

Here, you can:
  * Ask questions about:
    * How to setup the container.
    * How to use the application.
    * Problems you might with your specific setup.
    * etc.
  * Share ideas.
  * Engage with other community members.

Welcome others and be open-minded. Remember, this is a community we build together 💪.

**Comment by iosifLuchian** [comment]
for home assitant integration is there a way to integrate it like a web card(iframe) or like a dashboard for non admin user?

**Comment by kvarclovag** [comment]
Remark: ProtonMail doesn't allow to create free account from container
